backend-api/template.yaml

```diff
@@ -703,6 +701,7 @@ Resources:
           Projection:
             NonKeyAttributes:
               - sessionState
+              - redirectUri
```

Comment:
Have you deployed from main and then redeployed with these changes? I am wondering whether adding this will make the deployment fail? If it does then we might have to do that 2 step dance with the sessions table

Reply:
I re-deployed my backend stack a few minutes ago and re-tested so I am not sure what the issue is?

Reply:
Was the stack you deployed fresh? I'm wanting to test a stack that previously just had sessionState as a NonKeyAttribute in it's sessionTable and then update that stack with an updated sessions table with the new NonKeyAttribute to see if that will cause a deployment fail. 

I ask because it happened to me yesterday when I was playing around with Dynamo

Reply:
Currently testing

Reply:
Thank you so much @jmooney-dd ⭐

Reply:
Have tested and the deployment failed.

What I did:
- Deployed a `backend-api` and `sts-mock` from main. This stack only has ONE non key attribute listed
- Using the same stack, I then tried to deploy the changes in this PR which has a second non key attribute listed, but it failed

Resource that failed to deploy: `SessionsTable`
Error:
Resource handler returned message: "Cannot update GSI's properties other than Provisioned Throughput and Contributor Insights Specification. You can create a new GSI with a different name." (RequestToken: 6948247f-d788-a7c7-0869-dd4ae45270c6, HandlerErrorCode: InvalidRequest)

Possible solution
You could rename the SessionsTable in this PR to effectively create a new one destroying the old (not a live service so this is fine). You could then raise a follow up PR to change the name back.

```
    Properties:
      TableName: !Sub ${AWS::StackName}-sessions-table-${Environment}-2
```

@anahickmann

Reply:
Thanks @jmooney-dd. When I deployed a test stack to test my code changes I had to change the table name to make it work as well. I will do the same when I merge.